recipes/amplify_accelerate_te_fp8/train.py

```diff
@@ -35,6 +36,14 @@
 @hydra.main(config_path="hydra_config", config_name="L0_sanity", version_base="1.2")
 def main(args: DictConfig):
     """Entrypoint."""
+    # add wandb logging on main process
+    if not torch.distributed.is_initialized() or torch.distributed.get_rank() == 0:
+        wandb.init(
+            project=args.wandb_init_args.project,
+            name=args.wandb_init_args.name,
+            config=OmegaConf.to_container(args, resolve=True, throw_on_missing=True),
+            mode=getattr(args.wandb_init_args, "mode", "online"),
+        )
```

Comment:
The Trainer class set this up if we didn't do it manually right? https://huggingface.co/docs/transformers/en/main_classes/trainer#transformers.Seq2SeqTrainingArguments.report_to
probably makes sense to make sure we have the config object attached though